src/bindings/c/tests/ov_tensor_test.cpp

```diff
@@ -157,4 +157,44 @@ TEST(ov_tensor, ov_tensor_data) {
 
     ov_shape_free(&shape);
     ov_tensor_free(tensor);
+}
+
+TEST(ov_tensor, ov_tensor_element_type_test) {
+    ov_element_type_e type_list[] = {
+        ov_element_type_e::BOOLEAN,
+        ov_element_type_e::BF16,
+        ov_element_type_e::F16,
+        ov_element_type_e::F32,
+        ov_element_type_e::F64,
+        ov_element_type_e::I4,
+        ov_element_type_e::I8,
+        ov_element_type_e::I16,
+        ov_element_type_e::I32,
+        ov_element_type_e::I64,
+        ov_element_type_e::U1,
+        ov_element_type_e::U4,
+        ov_element_type_e::U8,
+        ov_element_type_e::U16,
+        ov_element_type_e::U32,
+        ov_element_type_e::U64,
```

Comment:
can it be a test parameter? I mean to create TEST_P and instantiate it then

Reply:
ok, will update PR soon.

Reply:
done